recipe/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 mkdir build
 cd build
+export CXXFLAGS="${CXXFLAGS} -DBOOST_TIMER_ENABLE_DEPRECATED=1"
```

Comment:
@junghans I guess this has to be addressed at some point but for now it is fine to ignore the warning.

Reply:
I think that is fixed in v2024.